docs/src/components/home/sections/HeroSection.tsx

```diff
@@ -102,7 +104,24 @@ export const HeroSection = () => {
             </Grid>
           ) : null}
 
-          <TerminalCommand command={frameworkInstallScript} variant="hero" />
+          {platform === 'flutter' ? (
+            <TerminalCommand command={frameworkInstallScript} variant="hero" />
+          ) : (
+            <Tabs>
+              <TabItem title="npm">
+                <TerminalCommand
+                  command={`npm install ${frameworkInstallScript}`}
```

Comment:
Should we use `npm i` or `npm install`? I do like `npm i` for its conciseness, but it's obviously subjective. 

Is there some official recommendation from main Amplify docs?

Reply:
I prefer npm install, feels more beginner friendly

Reply:
Converted all the `npm i` to `npm install` on the docs site